server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/Method.scala

```diff
@@ -1,30 +1,31 @@
-/****************************************************************
+/** **************************************************************
```

Comment:
why these changes on the license header?

Reply:
Maybe some miss click i did not recognize, i will fix it when rebase master for the Y work.